packages/ui-contract-editor/src/lib/styles.css

```diff
@@ -6,17 +6,41 @@ a {
     background-color: #e98181;
 }
 
-.conditional {
-    cursor: pointer;
+.variable,
+.conditional,
+.optional {
+    border-radius: 2px;
+    background-color: #fff;
+    color: #000;
+    padding: 0 3px 1px 3px;
```

Comment:
Did we want to change this to have the same top & bottom values? I think we were experimenting with `1px 3px` on our call.

Reply:
Ah yes!! 🙏 good catch